vitorsaa2k/HighestGoodNetworkApp
# Vitor fix save changes warning disappear

fixes bug (PRIORITY LOW) - 12 from USER PROFILE COMPONENT

## Description
When making changes to a user's profile, a reminder alert to click the save button is displayed. But even if the changes are undone before saving, the reminder alert remains on screen
 a. The reminder alert should disappear in case all changes on the user's profile are undone.

## Mainly changes
1. Now, the warning disappear when the user mannualy go back on the changes.
2. there was an optmization when updating the `updatedTasks` state.

## How to test
1 - check into current branch
2 - do npm install and ... to run this PR locally
3 - log as admin user
4 - Go to dashboard → User Profile → Change and go back to original on all the possible changable inputs.

## Example of how to test

https://user-images.githubusercontent.com/17365161/222005541-e84e979c-cd45-40ac-8709-91c11bcd1bf3.mp4
